mysql-test/suite/galera/r/versioning_trx_id.result

```diff
@@ -0,0 +1,21 @@
+connection node_1;
+create table t1 (a int, s bigint unsigned as row start, e bigint unsigned as row end, period for system_time(s,e)) engine=InnoDB with system versioning;
+insert into t1 (a) values (1),(2);
+connection node_2;
+insert into t1 (a) values (3),(4);
+select count(*) from mysql.transaction_registry where begin_timestamp='0-0-0';
```

Comment:
All your tests are negative, how I can see from these that something really replicates ?

Reply:
Fixed.

Reply:
Maybe I do not follow how this works, but I would like to see some positive selects that will show that contents of transaction_registry is really replicating from node_1 to node_2 i.e. there is really a result set and that actual time based query also returns a result set not only empty set.